public/app/features/alerting/unified/components/receivers/ReceiversTable.tsx

```diff
@@ -345,19 +337,20 @@ export const ReceiversTable = ({ config, alertManagerName }: Props) => {
     );
   }, [grafanaNotifiers.result, config.alertmanager_config, receiversMetadata]);
 
+  const [createSupported, createAllowed] = useAlertmanagerAbility(AlertmanagerAction.CreateContactPoint);
+
+  const [canReadSecrets] = useAlertmanagerAbility(AlertmanagerAction.CanReadSecrets);
```

Comment:
I believe it should be `[_, canReadSecrets]` 
The first variable tells if the operation is supported for the selected alertmanager, and the second one tells if it's permitted

Reply:
This also seems to be a duplicate of what we have in line 299

Reply:
fixed!